datasets/us/dc/med_exclusions/us_dc_med_exclusions.yml

```diff
@@ -0,0 +1,104 @@
+title: US DC Medicaid Sanctioned Providers
+entry_point: crawler.py
+prefix: us-meddc
+coverage:
+  frequency: daily
+  start: "2025-04-28"
+load_statements: true
+summary: >
+  List of providers excluded from participating in the District of Columbia Medicaid program.
```

Comment:
It's not just medicaid - I phoned and medicaid is included, but this is all providers barred from doing business with the district.

So let's make sure the metadata, prefix, and the dataset name are in line with their naming and not medicaid specific.